W-19811482 added environment variable instrux and replaced canvas scr… (#508)
mulesoft/docs-code-builder: modules/ROOT/pages/af-get-started.adoc

|===

== Verify and Add the Anypoint Environment Variable for Your Region

> Before you start Anypoint Code Builder desktop to create agent networks, make sure that the ANYPOINT_ENV environment variable is set to the appropriate value for your region. If you've already done this, you don't need to do it again.

Reviewer: ```suggestion
Before you start Anypoint Code Builder desktop to create agent networks, make sure that the `ANYPOINT_ENV` environment variable is set to the appropriate value for your region.
```

Reviewer: If the customers has to ensure doing x task, there's no need to repeat it again "If you've already done this, you don't need to do it again".
Is there a reason for this emphasis?

Reviewer: (Globally) - Ensure the appropriate format of "ANYPOINT_ENV" in "`ANYPOINT_ENV` environment variable".

**Sample Valid Values**

These are valid values for some geographic regions.
[cols="1,1"]
|===
|Region |Value 

|U.S.
|`prod`

|Canada
|`ca1`

|Europe
|`prod-eu`

|Japan
|`jp1`
|===

> === Linux and macOS

Reviewer: Ensure that ACB supports Linux.

> If you run Anypoint Code Builder desktop on Linux or macOS, follow these steps.

Reviewer: ACB does not support Linux. See https://docs.mulesoft.com/anypoint-code-builder/acb-release-notes#:~:text=Java%20versions.-,Operating%20system,-Desktop%20IDE

Author: Updated

> If you run Anypoint Code Builder desktop on Linux or macOS, follow these steps.

Reviewer: ```suggestion
If you run Anypoint Code Builder desktop on Linux or macOS, see [title X] and [title Y].
```

Reviewer: Actually, there are no steps and the reader may think that some docs are missing.

==== Verify the ANYPOINT_ENV Environment Variable

> . Open a Terminal window and run this command: `bash`.

Reviewer: ```suggestion
. Open a terminal window and run this command: `bash`.
```

Reviewer: I don't think "terminal" should take capital "T".

Author: I followed the same capitalization as I saw in ACB existing doc. Example: https://docs.mulesoft.com/anypoint-code-builder/tut-af-debug-am-flights#test

. Run this command: `echo $ANYPOINT_ENV`.

> If the value doesn't reflect your geographic region (or is empty), set the `ANYPOINT_ENV` environment variable.

Reviewer: ```suggestion
If the value doesn't reflect your geographic region, or it is empty, set the `ANYPOINT_ENV` environment variable.
```

==== Add the ANYPOINT_ENV Environment Variable

> Environment variables are typically added by adding an export command to a shell configuration file.

Reviewer: ```suggestion
Include environment variables by adding an export command to a shell configuration file.
```

> . Open a Terminal window.

Reviewer: ```suggestion
. Open a terminal window.
```

. Determine the shell configuration file you need to edit. To see your default shell, run `echo $SHELL`. Common files are:
** `~/.bashrc` or `~/.bash_profile` for Bash users
** `~/.zshrc` for Zsh users
** `~/.profile`
. Open the shell configuration file with a text editor. 
. In the text editor, at the end of the file, add this line: `export ANYPOINT_ENV={region value}`.
. Save and close the file.
. If you want to use the updated shell configuration file without logging out and back in, source the file. For example:
----
Bash
source ~/.zshrc  # Use the file you edited
----

=== Windows

> If you run Anypoint Code Builder desktop on Windows, follow these steps.

Reviewer: ```suggestion
If you run Anypoint Code Builder desktop on Windows, see [title X] and [title Y].
```

Author: future

==== Verify the ANYPOINT_ENV Environment Variable

. Open a Command Prompt or PowerShell window.
. If you're in a Command Prompt window, run this command: `echo %ANYPOINT_ENV%`. If you're in a PowerShell window, run this command: `echo $env:ANYPOINT_ENV`. 

If the value doesn't reflect your geographic region (or is empty), set the `ANYPOINT_ENV` environment variable.

==== Add the ANYPOINT_ENV Environment Variable Using System Properties

. Open System Properties. Tip: From the **Start** menu, search for `Edit the system environment variables`.
. In System Properties, click **Environment Variables**.
. Choose whether to set a User variable (for the current user) or a System variable (for all users). For most cases, setting a User variable is sufficient.
. In the User or System variable area, click **New**.
. In variable name, enter `ANYPOINT_ENV`.
. In variable value, enter the value for your region. For example, if you're in the U.S., enter `prod`.
. Save the changes.
. Open a new Command Prompt or PowerShell window to load the new variable.

==== Set the ANYPOINT_ENV Environment Variable Using Command Prompt or PowerShell

> If the `ANYPOINT_ENV` environment variable already exists, but you want to change the value quickly, use a Command Prompt or PowerShell window.

Reviewer: ```suggestion
If the `ANYPOINT_ENV` environment variable exists, and you want to change the value quickly, use a Command Prompt or PowerShell window.
```
